packages/app-store/routing-forms/lib/getResponseToStore.ts

```diff
@@ -19,6 +19,7 @@ export const getResponseToStore = ({
 
     response[field.id] = {
       label: field.label,
+      identifier: field?.identifier || null,
```

Comment:
This was generating the records to store. The field `identifier` wasn't being passed. All form fields should have an `identifier`